modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -242,25 +90,34 @@ Also consider to create a separate partition for [path]``/srv`` where Kiwi image
 | VirtIO Disk 3        | pgsql     | 100{nbsp}GB
 | VirtIO Disk 4        | swap      | 4{nbsp}GB
 |===
++
 
-. Click btn:[Add Hardware] to attach a virtual CDROM device with the prepared [systemitem]``Ignition`` or [systemitem]``Cloud Init`` disk.
 . Click btn:[Begin Installation] to boot the new VM from the {productname} image.
-  Wait until the login prompt is presented.
-  Log in using credentials set by configuration disk.
+. At the JeOS Firstboot screen select btn:[Start] to continue and check the following configuration options (keyboard layout, license agreement, time zone, password for root).
+
+. When installation completes, log in as root.
+
+. Proceed with xref:installation-and-upgrade:install-vm.adoc#minimal.susemgr.prep[].
 
 
-== {productname} Virtual Machine Settings - VMware
+
+// FIXME: check the following and esp. replace with firstboot (see 5.0 docs)
+== {productname} Virtual Machine Creation - VMware
 
 This sections describes VMware configurations, focusing on the creation of an extra virtual disk essential for the {productname} storage partition within VMware environments.
 
 .Procedure: Creating the VMware Virtual Machine
 
-. Download {productname} Server [filename]``.vmdk`` file then transfer a copy to your VMware storage.
+. Download {productname} Server [filename]``.vmdk`` file.
+  Then transfer a copy to your VMware storage.
 
-. Upload the prepared Ignition or Cloud Init disk file you created using the instructions above.
+. Make a copy of uploaded [filename]``.vmdk`` file using VMware web interface.
+  This will convert provided [filename]``.vmdk`` file to the format suitable for vSpere hypervisor.
```

Comment:
Maybe add clarification `Use this new copy as a base image for the virtual machine`